src/screens/extension-background-request/account-connect/account-connect.tsx

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import { View, ScrollView } from 'react-native';
+import { Button, Text } from '../../../library';
+import stylesProvider from './styles';
+import { smartConnect } from '../../../core/utils/smart-connect';
+import { IThemeProps, withTheme } from '../../../core/theme/with-theme';
+import { translate } from '../../../core/i18n';
+import bind from 'bind-decorator';
+import { IReduxState } from '../../../redux/state';
+import { IAccountState, IWalletsState, IWalletState } from '../../../redux/wallets/state';
+import { connect } from 'react-redux';
+import { BASE_DIMENSION } from '../../../styles/dimensions';
+import {} from 'react-native-gesture-handler';
```

Comment:
never seen this type of import - I am missing something? :)

Reply:
I don't know why it ended like that, maybe I was using something from `react-native-gesture-handler` and removed it afterwards. I think it's safe to just delete it.